### ISSUE TITLE
[doc] 4.5.2  Deploying a 389 Directory Server with an External Certificate

### ISSUE THREAD

**Issue by tgeek77** [SUSE/doc-caasp#605]
4.5.2  Deploying a 389 Directory Server with an External Certificate

https://documentation.suse.com/suse-caasp/4/single-html/caasp-admin/#_deploying_a_389_directory_server_with_an_external_certificate

Step 2: 

> Copy the external certificate <rootCA.pem> to a mounted data directory <DATA_DIR>/config/.

Don't you also need to import your key and certificate at the same time?

Step 7:

> Replace <KEY.PEM> with the input key.
> Replace <CERT.PEM> with the input certificate.

If you don't import your own <KEY.PEM>  and <CERT.PEM> in Step 2, then where do they come from?

Because that key and cert are not defined, I used my own. When I run the command:
`pk12util -i <CERT.PK12> -d /etc/dirsrv/slapd-<INSTANCE_NAME>`

I get:

```
pk12util -i cert.pk12 -d /etc/dirsrv/slapd-smt
Enter Password or Pin for "NSS Certificate DB":
```
This isn't the same password that I use for everything so it must have been generated when the container was first created. How do we find the password for this?

**Comment by r0ckarong** [comment]
@cclhsu Is this also covered in #603?

**Comment by cclhsu** [comment]
Yes, it is in #603, but I just closed that PR since there is 4.0.3 release plan to have the new 389-ds image, and it should go with #513.
